internal/consensus/replay_test.go

```diff
@@ -1217,6 +1217,9 @@ func (bs *mockBlockStore) PruneBlocks(height int64) (uint64, error) {
 	bs.base = height
 	return pruned, nil
 }
+func (bs *mockBlockStore) Close() error {
+	return nil
+}
```

Comment:
Optional cosmetic suggestion. 
```suggestion

func (bs *mockBlockStore) Close() error { return nil }
```

Just for basic legibility.